index.bs

```diff
@@ -2,7 +2,7 @@
 Title: Overview of the ResearcherPod specifications
 Shortname: overview
 Level: 1
-Status: LS
+Status: DREAM
```

Comment:
Can we at least make it a Living Document; these serve as project deliverables ;)

Reply:
Ok!